appveyor.yml

```diff
@@ -0,0 +1,335 @@
+            cp (Join-Path $env:APPVEYOR_BUILD_FOLDER 'releasenotes.txt') $binPackage
+            cscript (Join-Path $env:APPVEYOR_BUILD_FOLDER 'Tools\showdown\showdown.wsf') (Join-Path $env:APPVEYOR_BUILD_FOLDER 'README.md') (Join-Path $binPackage 'readme.html')
+
+            $cfgTemplates = (Join-Path $binPackage 'Configuration_Templates')
+            mkdir $cfgTemplates
+            cp (Join-Path $env:APPVEYOR_BUILD_FOLDER 'src\NHibernate.Config.Templates\*') $cfgTemplates
+
+            $reqBin = (Join-Path $binPackage 'Required_Bins')
+            mkdir $reqBin
+            cp (Join-Path $env:APPVEYOR_BUILD_FOLDER "src\NHibernate\bin\$env:CONFIGURATION\$env:targetFx\*") $reqBin -recurse
+            cp (Join-Path $env:APPVEYOR_BUILD_FOLDER "src\NHibernate\*.xsd") $reqBin
+
+            $tests = (Join-Path $binPackage 'Tests')
+            mkdir $tests
+            $vbBin = (Join-Path $env:APPVEYOR_BUILD_FOLDER "src\NHibernate.Test.VisualBasic\bin\$env:CONFIGURATION\$env:targetFx")
+            cp (Join-Path $vbBin 'DbScripts') $tests -recurse
+            cp (Join-Path $vbBin "NHibernate.Test.*") $tests
+            cp (Join-Path $vbBin "NHibernate.DomainModel.*") $tests
+            cp (Join-Path $vbBin "nunit*") $tests
+            cp (Join-Path $vbBin "*.xml") $tests -Exclude 'NHibernate.xml'
+            cp (Join-Path $vbBin 'log4net.dll') $tests
+            cp (Join-Path $env:APPVEYOR_BUILD_FOLDER "src\NHibernate.TestDatabaseSetup\bin\$env:CONFIGURATION\$env:targetFx\NHibernate.TestDatabaseSetup.*") $tests
+
+            cd $env:APPVEYOR_BUILD_FOLDER
+            7z a NHibernate-5.0.3-bin.zip NHibernate-5.0.3-bin\
+
+            # Reference package
+            choco install html-help-workshop
+            choco install sandcastle
+            $refPackage = (Join-Path $env:APPVEYOR_BUILD_FOLDER 'NHibernate-5.0.3-reference')
+            mkdir $refPackage
+            cp (Join-Path $env:APPVEYOR_BUILD_FOLDER 'doc\LICENSE.txt') $refPackage
+            #Common variables
+            $refFolder = (Join-Path $env:APPVEYOR_BUILD_FOLDER 'doc\reference')
+            $saxonClassPath = 'support/lib/avalon-framework-cvs-20020806.jar', 'support/lib/batik.jar', 'support/lib/fop-0.20.5-RFC3066-patched.jar', `
+                'support/lib/jai_codec.jar', 'support/lib/jai_core.jar', 'support/lib/rowan-0.1.jar', 'support/lib/saxon.jar', 'support/lib/saxon-dbxsl-extensions.jar'
+            for ($i=0; $i -lt $saxonClassPath.length; $i++) {
+                $saxonClassPath[$i] = (Join-Path $refFolder $saxonClassPath[$i])
+            }
+            $saxonClassPath = $saxonClassPath -Join ';'
+            $masterFile = (Join-Path $refFolder 'master.xml')
+            #Html
+            $html = (Join-Path $refPackage 'html')
+            mkdir $html
+            cd $html
+            java -classpath $saxonClassPath com.icl.saxon.StyleSheet $masterFile (Join-Path $refFolder 'styles/html_chunk.xsl')
```

Comment:
These java commands does also yield non-error messages in the error output, which is a bit annoying when watching the log output on AppVeyor.